scanomatic/data_processing/pheno/state.py

```diff
@@ -50,10 +50,15 @@ def serialized(self) -> list[Union[str, int, float]]:
             self.linear_regression_size,
             None if self.phenotypes_inclusion is None
             else self.phenotypes_inclusion.name,
-            self.no_growth_monotonicity_threshold,
-            self.no_growth_pop_doublings_threshold
+            None if self.no_growth_monotonicity_threshold is None
+            else self.no_growth_monotonicity_threshold,
+            None if self.no_growth_pop_doublings_threshold is None
+            else self.no_growth_pop_doublings_threshold,
         ]
 
+    def asdict(self) -> dict[str, Any]:
+        return asdict(self)
```

Comment:
this is useless, remove

Reply:
I tried using the `jsonizerd.dump`, but how the jsonizer needs to be extended/generalized to accommodate this type of object seemed to be non-trivial. I agree it should be done, but I would like to discuss how, before I set off in the wrong direction.

Reply:
fixed